src/test/kotlin/sh/hsp/techtree/graphviz/DSLConverterTest.kt

```diff
@@ -7,13 +7,12 @@ import sh.hsp.techtree.TreeNode
 
 class DSLConverterTest {
 
+    private val sut = DSLConverter()
```

Comment:
Suuuuty :XD
Ja tam wolę je nazywać po imieniu typu dslConverter - ale to od konwencji zależy.
Tutaj bym proponował jednak dslConverter